components/polls/DisplayPollResults.tsx

```diff
@@ -11,194 +12,181 @@ import {
+                <Text>{timerDisplay}</Text>
             </Center>
-
         </Tooltip>
     )
 }
 
-const ShieldBlock: React.FC<DisplayPollResultsProps> = ({pollData, totalVotes, onCountdownComplete}) => {
+// Component to display that a poll is currently locked.
+const ShieldBlock: React.FC<ShieldBlockProps> = ({ 
+    totalVotes,
+    countdown,
+    endTime
+}) => {
 
     return (
         <Center>
-        <div id="parent">
-            <Box style={{width:'100%', height:'10%', backgroundColor: 'blue', zIndex: '10', position: 'absolute'}}>
-                <Center><Text as='b' color='white'>Result Shielded</Text></Center>
-            </Box>
-
-            <Box style={{width:'100%', height:'100%', zIndex: '10', position: 'absolute'}}>
-                <Center style={{position: 'relative', top:'14%',  paddingLeft: 'auto', paddingRight: 'auto'}}>
-                    <Text as='b'>Total votes</Text>
-                </Center>
-                <Center style={{position: 'relative', top:'14%',  paddingLeft: 'auto', paddingRight: 'auto'}}>
-                    <Text as='b' fontSize='3xl'>{totalVotes}</Text>
-                </Center>
-                <Center style={{position: 'relative', top:'15%',  paddingLeft: 'auto', paddingRight: 'auto'}}>
-                    <Text as='b'>Unlocks in</Text>
-                </Center>
-                <Center style={{position: 'relative', top:'17%',  paddingLeft: 'auto', paddingRight: 'auto'}}>
-                    <Text as='b' fontSize='3xl'><Timer pollData={pollData} onCountdownComplete={onCountdownComplete}/></Text>
-                </Center>
-            </Box>
-            <div id="shield">
+            <div id="parent">
+                <Box style={{ width: '100%', height: '10%', backgroundColor: 'blue', zIndex: '10', position: 'absolute' }}>
+                    <Center><Text as='b' color='white'>Result Shielded</Text></Center>
+                </Box>
+                <Box style={{ width: '100%', height: '100%', zIndex: '10', position: 'absolute' }}>
+                    <Center style={{ position: 'relative', top: '14%', paddingLeft: 'auto', paddingRight: 'auto' }}>
+                        <Text as='b'>Total votes</Text>
+                    </Center>
+                    <Center style={{ position: 'relative', top: '14%', paddingLeft: 'auto', paddingRight: 'auto' }}>
+                        <Text as='b' fontSize='3xl'>{totalVotes ?? (<Spinner />)}</Text>
+                    </Center>
+                    <Center style={{ position: 'relative', top: '15%', paddingLeft: 'auto', paddingRight: 'auto' }}>
+                        <Text as='b'>Unlocks in</Text>
+                    </Center>
+                    <Center style={{ position: 'relative', top: '17%', paddingLeft: 'auto', paddingRight: 'auto' }}>
+                        <Text as='b' fontSize='3xl'>
+                            { countdown && endTime ? <Timer countdown={countdown} endTime={endTime} /> : (<Spinner size="3xl" />) }
```

Comment:
Could it be that we need to add a few ms / seconds to the timer to make sure the result is available when it runs out?